### ISSUE TITLE
[UIDS] Aria-label prop is broken on the MoneyInput

### ISSUE THREAD

**Issue by Dchyk** [user-interviews/ui-design-system#969]
If you pass `ariaLabel` as a prop to the MoneyInput, the underlying CurrencyInput component renders it as `arialabel` instead of `aria-label` meaning it doesn't work :-(

![Screenshot 2023-07-12 at 2 24 07 PM](https://github.com/user-interviews/ui-design-system/assets/25135946/cff8e23a-3c9e-4a16-9ee5-d1a5ccb088a2)

![Screenshot 2023-07-12 at 2 23 56 PM](https://github.com/user-interviews/ui-design-system/assets/25135946/bb1cc0fe-ca03-4e92-85e2-57953b464659)
